### PR TITLE
Augment Reference Manual

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Slight change in 'notify'

### CODE REVIEW

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me

File: taipy/gui/partial.py [hunk 1]
```diff
@@ -13,11 +13,17 @@ class Partial(_Page):
     and not related pages. This allows not to have to repeat yourself when
     creating your page templates.
 
-    Visual elements such as [`dialog`](../gui/viselements/dialog.md) or
+    Visual elements such as [`part`](../gui/viselements/part.md),
+    [`dialog`](../gui/viselements/dialog.md) or
```
Comment (FredLL-Avaiga) [inline taipy/gui/partial.py:17]: Are we missing expandable here?

**Review by FredLL-Avaiga — APPROVED** [review]
🐬

File: taipy/gui/partial.py (hunk 1 above)
Comment (FabienLelaquais) [reply taipy/gui/partial.py:17]: I just say...'such as'. All elements are not listed here, and you are right.

Comment (FabienLelaquais) [reply taipy/gui/partial.py:17]: And I wanted to stress on part now that... it works.

**Pull request merged**